### ISSUE TITLE
[gatsby-plugin-offline] users sometimes redirected to /offline-plugin-app-shell-fallback/index.html

### ISSUE THREAD

**Issue by endymion1818** [gatsbyjs/gatsby#7993]
Hi
We're using gatsby-plugin-offline to great effect, however sometimes users are redirected to /offline-plugin-app-shell-fallback/index.html even when they are connected.

Could you explain the navigateFallback option? I'd happily submit a PR for the readme of that plugin, but I don't understand it myself yet.

**Comment by KyleAMathews** [comment]
Are you on the latest v2 version of gatsby-plugin-offline?

**Comment by endymion1818** [comment]
Ah, I can't check that just now, don't have the work laptop on me. I'll check on Monday morning - I wasn't expecting you to be able to reply so soon Kyle! That's what happens when you delete ~400 old issues I guess ;-)

**Comment by dbrookes** [comment]
Any chance you're using the `gatsby-plugin-remove-trailing-slashes` plugin? If so then it's started happening recently (since workbox) if a user refreshes a cached offline page:

https://github.com/gatsbyjs/gatsby/issues/7869

**Comment by davidbailey00** [comment]
Closing, since this is a duplicate of #7869 - once PR #7943 is merged this should be fixed! (Sorry for not testing my earlier changes with this plugin)
